packages/gatsby-source-wordpress/src/utils/gatsby-version.ts

```diff
@@ -0,0 +1,5 @@
+import semver from "semver"
+const gatsbyVersion = require(`gatsby/package.json`)?.version
+
+// gt = greater than
+export const usingGatsbyV4OrGreater = semver.gt(gatsbyVersion, `4.0.0`)
```

Comment:
Semver is fine too, _CFLAGS_ can get compiled away in the future. It makes it easier to find and replace. Every major we will do a publish and bump peerdeps so technically >= will be incorrect anyways.

You can move to Number syntax if you want.

Reply:
@wardpeet I'm going to take over this PR to get it merged while Tyler is away. Are you saying that it should be `Number(_CFLAGS_.GATSBY_MAJOR) === 4` rather than `Number(_CFLAGS_.GATSBY_MAJOR) >= 4`. I'm not sure I understand why the next `_CFLAGS_.GATSBY_MAJOR` won't be `'5'`.